skopt/benchmarks.py

```diff
@@ -1,8 +1,42 @@
-"""A collection of benchmark problems"""
+"""A collection of benchmark problems."""
 
 import numpy as np
 
 
+def bench1(x):
+    """A benchmark function for test purposes.
+
+        f(x) = x ** 2
+
+    It has a single minima with f(x*) = 0 at x* = 0.
+    """
+    return np.asscalar(x ** 2)
+
+
+def bench2(x):
+    """A benchmark function for test purposes.
+
+        f(x) = x ** 2 if x < 0
```

Comment:
insert some spaces before the if maybe? Found it a bit tricky to find the `if` after spotting the `othewise`

L20 othewise -> otherwise